src/ClientData/FunctionInfo.cpp

```diff
@@ -18,7 +18,9 @@ uint64_t FunctionInfo::GetPrettyNameHash() const {
   return XXH64(pretty_name_.data(), pretty_name_.size(), 0xBADDCAFEDEAD10CC);
 }
 
-uint64_t FunctionInfo::FileOffset(uint64_t load_bias) const { return address() - load_bias; }
+uint64_t FunctionInfo::ComputeFileOffset(const ModuleData& module) const {
+  return module.ConvertFromVirtualAddressToOffsetInFile(address());
```

Comment:
Maybe we should also rename it to virtual address at some point, but not in this PR

Reply:
Added comment.

I will give more color just for completeness. Sometimes I wonder if we use nomenclature correctly or even if they had made up their mind in the spec. "Virtual address" is used both in ELF and PE specification. They seem to usually use it like we do, to refer to the address specified in the file. But I think the intended meaning of "virtual" is in contrast to "physical", which is something we are never interested in. We use "virtual" in contrast to "absolute", i.e., virtual address specified in the file vs memory virtual address where the function is loaded. One sentence from the ELF specification in particular makes me think this: "a file's virtual addresses might not match the memory virtual addresses during execution."

Why am I saying all this? Because if "virtual" is interpreted in contrast to "physical", then "virtual address" doesn't add much to "address". I think that's why historically sometimes we use "address" instead of "virtual address".

Anyway, we should stick to our "virtual" vs "absolute" naming as by now we are quite consistent with it, so changing "address" to "virtual address" in the various places where we still use the former is something we should probably do eventually.

Reply:
Another idea could be `preferred_address`. This would be inspired by `ImageBase` of PEs: "The preferred address of the first byte of image when loaded into memory": if that preferred addressed is satisfied, all the addresses specified in the file then equal the absolute addresses.